src/api/index.ts

```diff
@@ -7,48 +7,31 @@
 
 import axios from 'axios'
 
-let api: any
 let baseUrl: string = process.env.API_ENDPOINT
 	? process.env.API_ENDPOINT
 	: 'https://jsonplaceholder.typicode.com'
```

Comment:
use .env file

Reply:
I did have a .env file but as I put in in .gitignore, it was hidden in the source now. The raw api is just a fallback for people that clone our repo but have no env var yet

Reply:
Accepted